tests/app/shared/issue-tables/issues-table-paginated-data.spec.ts

```diff
@@ -0,0 +1,97 @@
+import { MatPaginator, MatPaginatorIntl } from "@angular/material";
+import { Issue } from "../../../../src/app/core/models/issue.model";
+import { TEAM_4 } from "../../../constants/data.constants";
+import {
+    ISSUE_WITH_ASSIGNEES,
+    ISSUE_WITH_EMPTY_DESCRIPTION,
+    ISSUE_WITH_EMPTY_DESCRIPTION_HIGH_SEVERITY,
+    ISSUE_WITH_EMPTY_DESCRIPTION_LOW_SEVERITY
+} from "../../../constants/githubissue.constants";
+import { setAndGetPaginatedData } from '../../../../src/app/shared/issue-tables/issue-tables-paginated-data';
+
+describe('isses-table-paginated-data', () => {
+    describe('getPaginatedData()', () => {
+        const changeDetectorRefMock = {
+            detectChanges: () => null,
+            markForCheck: () => null,
+            detach: () => null,
+            reattach: () => null,
+            checkNoChanges: () => null
+        };
+        let dataSet_3: Issue[];
+        let dataSet_7: Issue[];
+        let paginator: MatPaginator;
+        const dummyTeam = TEAM_4;
+        const mediumSeverityIssueWithResponse: Issue = Issue.createPhaseTeamResponseIssue(ISSUE_WITH_EMPTY_DESCRIPTION, dummyTeam);
+        const mediumSeverityIssueWithAssigneee: Issue = Issue.createPhaseTeamResponseIssue(ISSUE_WITH_ASSIGNEES, dummyTeam);
+        const lowSeverityFeatureFlawIssue: Issue = Issue.createPhaseTeamResponseIssue(ISSUE_WITH_EMPTY_DESCRIPTION_LOW_SEVERITY, dummyTeam);
+        const highSeverityDocumentationBugIssue: Issue =
+            Issue.createPhaseTeamResponseIssue(ISSUE_WITH_EMPTY_DESCRIPTION_HIGH_SEVERITY, dummyTeam);
+
+        beforeEach(() => {
+            dataSet_3 = [
+                mediumSeverityIssueWithResponse,
+                mediumSeverityIssueWithAssigneee,
+                lowSeverityFeatureFlawIssue
+            ];
+            dataSet_7 = [
+                mediumSeverityIssueWithResponse,
+                mediumSeverityIssueWithAssigneee,
+                lowSeverityFeatureFlawIssue,
+                mediumSeverityIssueWithResponse,
+                mediumSeverityIssueWithAssigneee,
+                lowSeverityFeatureFlawIssue,
+                highSeverityDocumentationBugIssue
+            ];
+            paginator = new MatPaginator(new MatPaginatorIntl(), changeDetectorRefMock);
+            paginator.pageSize = 3;
+        });
+
+        it('should set the length of paginator to the length of data', () => {
+            setAndGetPaginatedData(paginator, dataSet_3);
+            expect(paginator.length).toEqual(3);
```

Comment:
Not sure if it is a good idea to use **magic numbers** here. 

Maybe it would be better to put it as `toEqual(dataSet_3.length)` ?

Reply:
Yes, definitely! 😅  Thanks for the suggestion